CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+It may be changed whenever a new committee is elected ("New constitutional committee and/or threshold").
+Likewise, the committee threshold (the fraction of committee `Yes` votes that are required to ratify governance actions) is not fixed and
+can also be varied by the governance action.
+This gives a great deal of flexibility to the composition of the committee.
+In particular, it is possible to elect an empty committee if the community wishes to abolish the constitutional committee entirely. Note that this is different from a state of no-confidence and still constitutes a governance system capable of enacting proposals.
+
+#### Term limits
+
+Each newly elected constitutional committee will have a term limit.
+The system will automatically enter a state of no-confidence when the term limit for the constitutional
+committee expires.  This limit is a governance protocol parameter, which specifies the maximum number of epochs
+during which the committee can ratify governance actions.  When the committee term limit expires, all governance
+actions will be **dropped** and the protocol enters a state of no confidence.
+This means that the committee should plan for its own replacement if it wishes to avoid disruption.
+
+The term limit will reset whenever the  "New constitutional committee and/or threshold" governance action is enacted, even if the same committee is re-elected
+and the threshold remains unchanged.  This allows Ada holders to confirm their confidence in the committee if they wish.
+Note that the term limit is calculated at the point the committee is elected.  Any change in the underlying protocol
+parameter only affects the term that applies to future committees, and does not change the term of the current committee.
+
+
+<!--------------------------- Constitutional committee ------------------------>
+<!---------------------------           DReps          ------------------------>
+
+### Delegated representatives (DReps)
+
+> **Warning**
+> CIP-1694 DReps **should not be conflated** with Project Catalyst DReps.
+
+<!-- TODO find another name that still points to liquid democracy. -->
+
+#### Pre-defined DReps
+
+In order to participate in governance, each stake credential must be delegated to a DRep.
+Ada holders will generally delegate their voting rights to a registered DRep
+that will vote on their behalf.  In addition, two pre-defined DRep options are available:
+
+* `Abstain`
+
+  If an Ada holder delegates to `Abstain`, then their stake is actively marked
+  as not participating in governance.
+
+  The effect of delegating to `Abstain` on chain is that the delegated stake *will not* be considered to be
+  a part of the active voting stake.  However, the stake *will* be considered to be registered for the
+  purpose of the incentives that are described in [Incentives for Ada holders to delegate voting stake](#incentives-for-ada-holders-to-delegate-voting-stake).
+
```

Comment:
```suggestion
  The effect of delegating to `Abstain` on chain is that the delegated stake *will not* be considered to be
  a part of the active voting stake.  However, the stake *will* be considered to be registered for the
  purpose of the incentives that are described in [Incentives for Ada holders to delegate voting stake](#incentives-for-ada-holders-to-delegate-voting-stake).


> **Note**
> Delegating to the pre-defined `Abstain` dRep can be viewed as **passive** abstention. The delegated user's stake is not counted towards active, participating stake in any governance action and, for all intents and purposes, the user's opinions on any governance matters is opaque. **This pre-defined dRep will not cast any vote during any governance action on behalf of the users delegating.**
>
> This is a fundamentally different form of "protest vote" from **active** abstention where a user may "cast their ballot" during a governance action but take no choice (i.e. "Vote Abstain") in the outcome. 
```

This was a major point of my misunderstanding based on the previous language. I would suggest we add this clarity explicitly to prevent future misunderstandings.

Reply:
I'll try to put this in my own words, I appreciate you letting us know what wasn't clear. We are not prescribing any intention behind abstention in general, I'd rather not pretend that there is such a thing as "passive" and "active" abstention. But I can make it clearer that the predefined DReps do not actually vote per say. People may have vastly different reasons for both delegating to the Abstain DRep and for voting abstain.

Reply:
It is a largely philosophical difference in my mind, but you can "make your voice heard" (i.e. get the "I voted" sticker) by logging an actually submitted ballot while making no choice which does differentiate from not submitting a ballot at all. I know it's probably beyond the scope of this document but should be kept in mind.

Reply:
From a mechanical perspective, an "active abstention" could be viewed as counting towards "participating active voting stake" (even though they do not affect the outcome thresholds) whereas the abstain dRep would count towards neither metric.